src/alembic.ini

```diff
@@ -0,0 +1,116 @@
+# A generic, single database configuration.
+
+[alembic]
+# path to migration scripts
+script_location = migrations
+
+# template used to generate migration file names; The default value is %%(rev)s_%%(slug)s
```

Comment:
clean the comments, it doesn't make sense to keep the template generated, we need to keep the comments that make sense for the project